src/pages/SetPasswordPage.js

```diff
@@ -117,6 +117,7 @@ class SetPasswordPage extends Component {
                                 message={error}
                                 isAlertVisible={!_.isEmpty(error)}
                                 isDisabled={!this.state.isFormValid}
+                                disablePressOnEnter
```

Comment:
Enter should submit the form, right? Can you clarify

Reply:
https://github.com/Expensify/App/blob/94057baaf27d1f6c28277fc5cd713ee42ec10ada/src/pages/SetPasswordPage.js#L103

Enter is handled in this component. `disablePressOnEnter` prevents attaching a global listener.